modules/kvantum/hm.nix

```diff
@@ -0,0 +1,41 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}: {
+  options.stylix.targets.kvantum.enable =
+    config.lib.stylix.mkEnableTarget "Kvantum" pkgs.stdenv.hostPlatform.isLinux;
+
+  config = lib.mkIf config.stylix.targets.kvantum.enable (let
+    kvconfig = config.lib.stylix.colors {
+      template = ./kvconfig.mustache;
+      extension = ".kvconfig";
+    };
+    svg = config.lib.stylix.colors {
+      template = ./kvantum-svg.mustache;
+      extension = "svg";
+    };
+    kvantumPackage = pkgs.runCommandLocal "base16-kvantum" {} ''
+      mkdir -p $out/share/Kvantum/Base16Kvantum
+      cat ${kvconfig} >>$out/share/Kvantum/Base16Kvantum/Base16Kvantum.kvconfig
+      cat ${svg} >>$out/share/Kvantum/Base16Kvantum/Base16Kvantum.svg
+    '';
+  in {
+    home.packages = with pkgs; [
+      qt5ct
+      libsForQt5.qtstyleplugin-kvantum
+      qt6Packages.qtstyleplugin-kvantum
+      kvantumPackage
```

Comment:
These should be added automatically by home-manager

Reply:
After trying this, I noticed that configuring the icon theme reliably becomes quite a lot harder, so I would like to keep this using qt5ct. If a method to set icon themes reliably without qt5ct is found, I'm open to make changes to this PR.